src/components/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -160,6 +160,33 @@ export default class ListPeopleAndRoles extends Mixins(CommonMixin) {
   /** Headers for the person table. */
   readonly tableHeaders: Array<string> = ['Name', 'Mailing Address', 'Delivery Address', 'Roles']
 
+  /**
+   * Checks if specified org/person was added.
+   * @param person The org/person to check.
+   * @returns True if the org/person was added.
+   */
+  private wasAdded (person: OrgPersonIF): boolean {
+    return (person.action === ActionTypes.ADDED)
+  }
+
+  /**
+   * Checks if specified org/person was edited.
+   * @param person The org/person to check.
+   * @returns True if the org/person was edited.
+   */
+  private wasEdited (person: OrgPersonIF): boolean {
```

Comment:
This is to allow (in the future) a more detailed comparison of new vs old (eg, for identifying what data was updated). But Scott said that's not needed right now so I went no further. But I kept this code anyway :)

Reply:
If this is the approach we want, i can move this functionality into the CHIP common component and make it generic in my PR. 
I just figured we would be applying this all over the place, so it made sense to create something.

Reply:
Sure (and I saw your other comment about the v-chip).

My explanation here was why I removed the "if" out of the template element and into the script code.